diktat-rules/src/main/kotlin/com/saveourtool/diktat/ruleset/rules/chapter2/kdoc/KdocMethods.kt

```diff
@@ -197,11 +201,42 @@ class KdocMethods(configRules: List<RulesConfig>) : DiktatRule(
                 && !hasReturnKdoc && !isReferenceExpressionWithSameName
     }
 
+    private fun isThrowInTryCatchBlock(node: ASTNode?): Boolean {
+        node ?: return false
+        var parent = node.treeParent
+        while (parent != null && parent.elementType != TRY) {
+            parent = parent.treeParent
+        }
```

Comment:
As far as I remember, there is a `node.parents()` function to find all the parents of the current node.

Reply:
Done